src/game/logic/object/object.cpp

```diff
@@ -23,10 +23,28 @@ ObjectShroudStatus Object::Get_Shrouded_Status(int index) const
     return ObjectShroudStatus(0);
 }
 
+// zh: 0x005479B0 wb: 0x007D0E3D
+Relationship Object::Get_Relationship(const Object *that) const
+{
+#ifdef GAME_DLL
+    return Call_Method<Relationship, const Object, const Object *>(PICK_ADDRESS(0x005479B0, 0x007D0E3D), this, that);
+#else
+    return Relationship();
```

Comment:
What about `that` argument? Should not be passed here?

Reply:
This is just to return a default value for the unimplemented function